spec/rpc/README.md

```diff
@@ -0,0 +1,248 @@
+# RPC spec
```

Comment:
TODO: We should update `rpc/openapi/openapi.yml`. I'll fix it later

Reply:
I'll make issues that be found in this PR later.